test/acceptance/features/steps/steps.py

```diff
@@ -118,7 +120,7 @@ def imported_nodejs_app_is_running(context, application_name):
 def imported_nodejs_app_is_not_running(context, application_name):
     namespace = context.namespace
     application = NodeJSApp(application_name, namespace.name)
-    application.is_running() | should.be_falsy.desc("Aplication not running")
+    application.is_running() | should.be_falsy.desc("Application not running")
```

Comment:
replace with assert

Reply:
I have replaced should with assert with corresponding error message.